### DISCUSSION TITLE
fluentd not removing pointer from old file

### DISCUSSION THREAD

**Discussion by mangeshmj1992** [fluent/fluent-bit#6124] [Q&A]
Hello team,
   We are using td agent bit to ship our logs to elk. Our Source file format contains date like : filename_2022_09_30_00_00_01.log , 2022_09_29_00_00_01.log.

that's why while configuring td agent bit we are using  filename_*. That's why td agent bit keep pointer for all files and when script trying to delete old files it is saying that file is in use.

We are deleting old files by stopping td agent bit but we need permanent solution on that.

Question:
1. is there any way to configure td agent bit path using todays date rather than *.
2. Is there any way to remove harvester from old file which are not updated from last 1 or 2 days.

**Comment by patrick-stephens** [comment]
Use a wildcard and the `ignore_older 1d` option for `tail`? There was a similar discussion on Slack: https://fluent-all.slack.com/archives/C0CTQGHKJ/p1665481847707309
